src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -560,17 +566,67 @@ function tryRemoveElement(elem) {
             return relativeOffset;
         }
 
+        /**
+         * @private
+         * These browsers will not clear the existing active cue when setting an offset
+         * for native TextTracks.
+         * Any previous text tracks that are on the screen when the offset changes will remain next
+         * to the new tracks until they reach the end time of the new offset's instance of the track.
+         */
+        requiresHidingActiveCuesOnOffsetChange() {
+            return !!browser.firefox;
+        }
+
+        /**
+         * @private
+         */
+        hideTextTrackWithActiveCues(currentTrack) {
+            if (currentTrack.activeCues) {
+                currentTrack.mode = 'hidden';
+            }
+        }
+
+        /**
+         * Forces the active cue to clear by disabling then re-enabling the track.
+         * The track mode is reverted inside of a 0ms timeout to free up the track
+         * and allow it to disable and clear the active cue.
+         * The track needs to be temporarily stored in order for us to access it
+         * inside the timeout. The stored value is reset after it is used.
+         * @private
+         */
+        forceClearTextTrackActiveCues(currentTrack) {
+            if (currentTrack.activeCues) {
+                this.#currentTextTrack = currentTrack;
+                currentTrack.mode = 'disabled';
+                setTimeout(() => {
+                    if (this.#currentTextTrack) {
+                        this.#currentTextTrack.mode = 'showing';
+                        this.#currentTextTrack = null;
+                    }
+                }, 0);
+            }
+        }
+
         /**
          * @private
          */
         setTextTrackSubtitleOffset(currentTrack, offsetValue) {
             if (currentTrack.cues) {
                 offsetValue = this.updateCurrentTrackOffset(offsetValue);
+                const shouldClearActiveCues = offsetValue && this.requiresClearingActiveCuesOnOffsetChange();
```

Comment:
```suggestion
                const shouldClearActiveCues = offsetValue && this.requiresHidingActiveCuesOnOffsetChange();
```

Reply:
That's embarrassing. I must have tried to change the name but must have not saved correctly or something.